config/migrations/sqlite3/6_create_namespaces.up.sql

```diff
@@ -0,0 +1,85 @@
+PRAGMA foreign_keys=off;
+
+/* Create namespaces table */
+CREATE TABLE IF NOT EXISTS namespaces (
+  key VARCHAR(255) PRIMARY KEY UNIQUE NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  description TEXT NOT NULL,
+  protected BOOLEAN DEFAULT FALSE NOT NULL,
+  created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+  updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+/* Create default namespace */
+INSERT INTO namespaces (key, name, description, protected) VALUES ('default', 'Default', 'Default namespace', true);
+
+/* SQLite doesn't allow you to drop unique constraints with ALTER TABLE
+   so we have to create a new table with the schema we want and copy the data over.
+   https://www.sqlite.org/lang_altertable.html
+*/
+
+/* Create temp tables */
+
+/* flags */
+CREATE TABLE IF NOT EXISTS flags_temp (
+  key VARCHAR(255) NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  description TEXT NOT NULL,
+  enabled BOOLEAN DEFAULT FALSE NOT NULL,
+  created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+  updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+  namespace_key VARCHAR(255) NOT NULL DEFAULT 'default' REFERENCES namespaces ON DELETE CASCADE,
+  PRIMARY KEY (namespace_key, key)); /* composite primary key unique by default */
+
+INSERT INTO flags_temp (key, name, description, enabled, created_at, updated_at)
+  SELECT key, name, description, enabled, created_at, updated_at
+  FROM flags;
+
+DROP TABLE flags;
+
+ALTER TABLE flags_temp RENAME TO flags;
```

Comment:
Genuine open question: Is there any value in breaking the creation of the namespaces table and the migration of the flags table into separate migrations? 🤔 

Maybe only for clarity. I doubt there is little functional value to us, but food for thought.